### ISSUE TITLE
Torch 2.0.1 and Torchvision 0.15.2 Version Requirement - Is it really this exact?

### ISSUE THREAD

**Issue by eugeneswalker** [Multilevel-NN/torchbraid#19]
Does torchbraid truly require exactly torch 2.0.1 and torchvision 0.15.2? Would it be reasonable to expect it to work with newer torch 2.2.1 and vision 0.17.2? Thanks!

**Comment by eric-c-cyr** [comment]
I think so. Though let me know if not!

If so we can adjust the pip install too

**Comment by eric-c-cyr** [comment]
I've recently been relaxing all of these things. Hopefully that has helped if you are still pursuing this?

**Comment by eugeneswalker** [comment]
> I've recently been relaxing all of these things. Hopefully that has helped if you are still pursuing this?

Yes! I've been patching this to relax constraints and seems to have worked fine, although haven't extensively tested functionality so there may be issues I haven't discovered. Thanks!

**Comment by eric-c-cyr** [comment]
It is now relaxed to be greater than 15.0.2. Closing and greater than torch>2.0..
